packages/core-cairo/src/print.ts

```diff
@@ -184,9 +185,11 @@ function printFunction(fn: ContractFunction): Lines[] {
 
   const returnArgs = fn.returns?.map(a => typeof a === 'string' ? a : a.name);
 
-  fn.libraryCalls.forEach(callFn => {
-    const libraryCall = `${getFunctionName(callFn)}()`;
-    code.push(libraryCall);
+  fn.libraryCalls.forEach(libraryCall => {
+    const libraryCallString = libraryCall.literalArgs ? 
+      `${getFunctionName(libraryCall.baseFn)}(${libraryCall.literalArgs.join(', ')})`:
+      `${getFunctionName(libraryCall.baseFn)}()`;
```

Comment:
`literalArgs` is always an array as far as I can tell.

```suggestion
    const libraryCallString = 
      `${getFunctionName(libraryCall.baseFn)}(${libraryCall.literalArgs.join(', ')})`;
```